smart-client-core/src/main/java/com/emc/rest/smart/SmartConfig.java

```diff
@@ -132,6 +132,12 @@ public Object getProperty(String propName) {
         return properties.get(propName);
     }
 
+    public Integer getPropertyIntValue(Object propertyValue, int defaultValue) {
```

Comment:
This method should follow the convention of `getProperty()` - so it should take in a property _**name**_, and return the default if it does not exist.

I would also rename this to `getIntProperty()`, as that is the convention used by other libraries.